server/views/formPages/incident/evidence.html

```diff
@@ -61,34 +58,29 @@ <h1 class="govuk-heading-xl mainHeading">Evidence</h1>
                         html: 'Tag number'
                       },
                       classes: 'govuk-!-width-one-third',
-                      id: 'tag-numbers[' + loop.index0 + '][name]',
-                      name: 'tagNumbers[' + loop.index0 + '][name]',
-                      value: tagNumber.name,
+                      id: 'evidence-tag-and-description[' + loop.index0 + '][evidenceTagReference]',
+                      name: 'evidenceTagAndDescription[' + loop.index0 + '][evidenceTagReference]',
+                      value: evidenceTagAndDescription.evidenceTagReference,
                       attributes: {
-                        'data-name': 'tagNumbers[%index%][name]',
-                        'data-id': 'tagNumbers[%index%][name]'
+                        'data-name': 'evidenceTagAndDescription[%index%][evidenceTagReference]',
+                        'data-id': 'evidence-tag-and-description[%index%][evidenceTagReference]'
                       }
                     })
                   }} 
-
-                  {% set index = loop.index0 %}
-
-                  <!--this assignment is required because can't access both tagNumber and evidenceDescriptions in same loop.
-                  The value for index will change on each iteration so will keep both data keys in sync -->
                             
```

Comment:
General comment about text areas: the cat tool uses govukCharacterCount which gives the user some guidance on whether there is too much input text

Reply:
We did discuss character count with the Andy Hudson, the designer. He is already looking at what the limit should be. Will implement when he responds.